tests/test_basic.py

```
@@ -78,6 +91,60 @@ def test_delete_task(self):
+        )
+        self.assertEqual(resp.status_code, 200)
+        image_path = os.path.join(
+            self.app.static_folder, 'subjects', secure_filename(subject) + '.png'
+        )
+        self.assertTrue(os.path.exists(image_path))
+        os.remove(image_path)
+
+    def test_task_has_dynamic_image_url(self):
+        """Aufgaben erhalten eine dynamische Pollinations-Bild-URL."""
+        with self.app.app_context():
+            task = self.app.task_manager.add_task('Hausaufgabe')
```

Ruff (0.12.2)

Local variable `task` is assigned to but never used

Remove assignment to unused variable `task`

(F841)